jx_sqlite/query_table.py

```diff
@@ -87,8 +87,8 @@ def query(self, query):
             Log.error("Expecting table, or some nested table")
         frum, query['from'] = query['from'], self
         schema = self.sf.tables["."].schema
-        query = QueryOp.wrap(query, schema)
-
+        if not query.groupby:
+            query = QueryOp.wrap(query, schema)                        
```

Comment:
these two lines can probably be moved to the main if-block starting at line 113.  Specifically, add `query = QueryOp.wrap(query, schema) ` to the beginning of each `elif` block